### ISSUE TITLE
exceljs/lib/xlsx/xform/sheet/hyperlink-map.js error: undefined

### ISSUE THREAD

**Issue by liuxue0905** [exceljs/exceljs#195]
```
var HyperlinkMap = function(relationships, hyperlinks) {
  var rels = {};
  var map = this.map = {};
  if (relationships) {
    relationships.forEach(function (relationship) {
      rels[relationship.rId] = relationship.target;
    });

    if (hyperlinks) { // HERE : In my case, hyperlinks was undefined.
      hyperlinks.forEach(function (hyperlink) {
        map[hyperlink.address] = rels[hyperlink.rId];
      });
    }
  }
};
```

```
Unhandled rejection TypeError: Cannot read property 'forEach' of undefined
    at Object.HyperlinkMap (/Users/liuxue/WebstormProjects/app/node_modules/exceljs/lib/xlsx/xform/sheet/hyperlink-map.js:33:15)
    at utils.inherits.reconcile (/Users/liuxue/WebstormProjects/app/node_modules/exceljs/lib/xlsx/xform/sheet/worksheet-xform.js:193:29)
    at /Users/liuxue/WebstormProjects/app/node_modules/exceljs/lib/xlsx/xlsx.js:104:22
    at Array.forEach (native)
    at Object.XLSX.reconcile (/Users/liuxue/WebstormProjects/app/node_modules/exceljs/lib/xlsx/xlsx.js:102:22)
    at /Users/liuxue/WebstormProjects/app/node_modules/exceljs/lib/xlsx/xlsx.js:198:16
    at tryCatcher (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/promise.js:507:31)
    at Promise._settlePromiseAt (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/promise.js:581:18)
    at Promise._settlePromiseAtPostResolution (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/promise.js:245:10)
    at Async._drainQueue (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/async.js:128:12)
    at Async._drainQueues (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/liuxue/WebstormProjects/app/node_modules/bluebird/js/main/async.js:15:14)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)
```

**Comment by stavenko** [comment]
Got same issue:

```
var HyperlinkMap = function(relationships, hyperlinks) {
  var rels = {};
  var map = this.map = {};
  hyperlinks = hyperlinks ? hyperlinks : []; // Quick fix
  if (relationships) {
    relationships.forEach(function (relationship) {
      rels[relationship.rId] = relationship.target;
    });

    hyperlinks.forEach(function (hyperlink) {
      map[hyperlink.address] = rels[hyperlink.rId];
    });
  }
};
```
After this fix everything works properly, but I need this to be fixes in npm registry.

**Comment by tkirda** [comment]
I think this has been fixed with #190
